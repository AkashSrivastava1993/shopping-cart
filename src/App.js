import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import Main from './components/Main';
import OrderPlaced from './components/OrderPlaced';
import NotFound from './components/NotFound';
function App() {

  return (
    <>
      <Router>
        <Switch>
          <Route exact path="/" component={Main} />
          <Route path="/ordered" component={OrderPlaced} />
          <Route path="/*" component={NotFound} />
        </Switch>
      </Router>
    </>
  );
}

export default App;