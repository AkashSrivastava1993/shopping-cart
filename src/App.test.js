import { render, screen } from '@testing-library/react';
import App from './App';
import React from 'react';
import '@testing-library/jest-dom';

it('renders', () => {
  const { asFragment } = render(<App/>);
  expect(asFragment()).toMatchSnapshot();
});

