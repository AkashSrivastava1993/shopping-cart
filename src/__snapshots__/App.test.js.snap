// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <header
      class="block row center"
    >
      <div>
        <a
          href="#/"
        >
          <h1>
            Shopping Cart
          </h1>
        </a>
      </div>
      <div>
        <a
          href="#/cart"
        >
          Cart 
        </a>
         
      </div>
    </header>
    <div
      class="row"
    >
      <main
        class="block col-2"
      >
        <h2>
          Products
        </h2>
        <input
          id="myInput"
          placeholder="Search for products..."
          type="text"
        />
        <div
          class="row"
        >
          <div>
            <img
              alt="MacBook"
              class="small"
              src="https://picsum.photos/id/180/2400/1600"
            />
            <h3>
              MacBook
            </h3>
            <div>
              ₹140000
            </div>
            <div>
              <button>
                Add To Cart
              </button>
            </div>
          </div>
          <div>
            <img
              alt="Car"
              class="small"
              src="https://picsum.photos/id/111/4400/2656"
            />
            <h3>
              Car
            </h3>
            <div>
              ₹240000
            </div>
            <div>
              <button>
                Add To Cart
              </button>
            </div>
          </div>
          <div>
            <img
              alt="Shoes"
              class="small"
              src="https://picsum.photos/id/21/3008/2008"
            />
            <h3>
              Shoes
            </h3>
            <div>
              ₹1800
            </div>
            <div>
              <button>
                Add To Cart
              </button>
            </div>
          </div>
        </div>
      </main>
      <aside
        class="block col-1"
      >
        <h2>
          Cart Items
        </h2>
        <div>
          <div>
            Cart is empty
          </div>
        </div>
      </aside>
    </div>
  </div>
</DocumentFragment>
`;
